content/docs/0.8.0/users-guide/default-disk-and-node-config.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Configuring Defaults for Nodes and Disks
+weight: 1
+---
+
+_Available as of v0.8.1_
+
+This feature allows the user to customize the default disks and node configurations in Longhorn for newly added nodes using Kubernetes labels and annotations instead of the Longhorn API or UI.
+
+Customizing the default configurations for disks and nodes is useful for scaling the cluster because it eliminates the need to configure Longhorn manually for each new node if the node contains more than one disk, or if the disk configuration is different for new nodes.
+
+Longhorn will not keep the node labels or annotations in sync with the current Longhorn node disks or tags. Nor will Longhorn keep the node disks or tags in sync with the nodes, labels or annotations after the default disks or tags have been created.
+
+### Adding Node Tags to New Nodes
+
+When a node does not have a tag, you can use a node annotation to set the node tags, as an alternative to using the Longhorn UI or API.
+
+1. Scale up the Kubernetes cluster. The newly added nodes contain no node tags.
+2. Add annotations to the new Kubernetes nodes that specify what the default node tags should be. The annotation format is:
+
+    ```
+    node.longhorn.io/default-node-tags: <node tag list with JSON string format>
+    ```
+    For example:
+    
+    ```
+    node.longhorn.io/default-node-tags: '["fast","storage"]'
+    ``` 
+3. Wait for Longhorn to sync the node tag automatically.
+
+> **Result:** If the node tag list was originally empty, Longhorn updates the node with the tag list, and you will see the tags for that node updated according to the annotation. If the node already had tags, you will see no change to the tag list.
+
+### Customizing Default Disks for New Nodes
+
+Longhorn uses the **Create Default Disk on Labeled Nodes** setting to enable default disk customization.
+
+- If the setting is enabled, Longhorn waits for the default disk customization to be set.
+- If the setting is disabled, Longhorn will create the default disk for any newly added nodes, using the disk path specified by settings `Default Data Path`. Then Longhorn relies on the value of the node label `node.longhorn.io/create-default-disk` to decide how to customize the default disks.
+
+The value and labels will be in effect only when the setting is enabled.
+
+If the value of the label is `config`, the annotation will be parsed and used as the default disk customization. 
+
+If the label is set to `true`, Longhorn will use `settings.default-data-path` to create a default disk if the node has no disks.
+
+Other values will be treated as false and no default disk will be created.
+
+The default disk will not be automatically created on the new nodes unless the `create-default-disk` label is set.
```

Comment:
This is not true if the setting was disabled.

Basically it's like this:
1. If the setting is disabled, Longhorn will create default disk using setting.default-data-path on all new nodes.
2. If the setting is enabled, Longhorn will decide creating the default disks or not, depends on the node's label value
    1. If the node's label value is `true`, Longhorn will create default disk using settings.default-data-path on the node.
    2. If the node's label value is `config`, Longhorn will check for the annotation and create default disks according to it
        1. if there is no annotation or annotation is invalid or the label value is invalid, Longhorn will not change anything.

Reply:
### Customizing Default Disks for New Nodes

Longhorn uses the **Create Default Disk on Labeled Nodes** setting to enable default disk customization.

If the setting is disabled, Longhorn will create a default disk using `setting.default-data-path` on all new nodes.

If the setting is enabled, Longhorn will decide to create the default disks or not, depending on the node's label value of `node.longhorn.io/create-default-disk`.
    
- If the node's label value is `true`, Longhorn will create the default disk using `settings.default-data-path` on the node. If the node already has existing disks, Longhorn will not change anything.
- If the node's label value is `config`, Longhorn will check for the `node.longhorn.io/default-disks-config` annotation and create default disks according to it. If there is no annotation, or if the annotation is invalid, or the label value is invalid, Longhorn will not change anything.

The value of the label will be in effect only when the setting is enabled.

If the `create-default-disk` label is not set, the default disk will not be automatically created on the new nodes.

The configuration described in the annotation only takes effect when there are no existing disks or tags on the node.

If the label or annotation fails validation, the whole annotation is ignored.

Reply:
```
If the create-default-disk label is not set, the default disk will not be automatically created on the new nodes when the setting is enabled.
```

Reply:
updated